### PR TITLE
fix our jsi18n files, clean-up old code

### PR DESCRIPTION
**Pull request by LeoMcA**
the jsi18n files were being compiled without the locale files present

remove last references to yaocho

remove (broken) dynamic jsi18n routes

https://github.com/mozilla/sumo-project/issues/780

### CODE REVIEW

File: kitsune/urls.py [hunk 1]
```diff
@@ -40,21 +38,6 @@
     url(r"^announcements", include("kitsune.announcements.urls")),
     url(r"^community", include("kitsune.community.urls")),
     url(r"^badges/", include("kitsune.kbadge.urls")),
-    # Javascript translations.
-    url(
```
Comment (akatsoulas) [inline kitsune/urls.py:44]: why are you removing this?

File: Dockerfile [hunk 2]
```diff
@@ -123,7 +126,6 @@ RUN groupadd --gid 1000 kitsune && useradd -g kitsune --uid 1000 --shell /usr/sb
 COPY --from=base --chown=kitsune:kitsune /venv /venv
 COPY --from=base --chown=kitsune:kitsune /vendor /vendor
 COPY --from=staticfiles --chown=kitsune:kitsune /app/static /app/static
-COPY --from=staticfiles --chown=kitsune:kitsune /app/jsi18n /app/jsi18n
```
Comment (akatsoulas) [inline Dockerfile:126]: why are you removing this line?

**Comment by ABETTERLIFE-CYBER** [comment]
What is it?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

File: kitsune/urls.py (hunk 1 above)
Comment (akatsoulas) [reply kitsune/urls.py:44]: discussed in mtg

File: Dockerfile (hunk 2 above)
Comment (akatsoulas) [reply Dockerfile:126]: discussed in mtg

**Review by akatsoulas — APPROVED** [review]

**Comment by akatsoulas** [comment]
there is a test checking for the removed endpoint

**Pull request merged**